conf/report/report-template.html

```diff
@@ -143,7 +143,14 @@ <h1 class="headline"><a href="https://github.com/chipsalliance/sv-tests">SV-Test
   <br> <br>
   <footer id="footer">
     Generated using <a href="https://github.com/chipsalliance/sv-tests">sv-tests</a>,
-    revision: <a href="https://github.com/chipsalliance/sv-tests/commit/{{revision}}">{{revision}}</a>.
+    revision: <a href="https://github.com/chipsalliance/sv-tests/commit/{{revision}}">{{revision}}</a>,
+    {% if build_id != 'local' %}
+      {% set href = https://github.com/chipsalliance/sv-tests/actions/runs/ + build_id %}
```

Comment:
This is the candidate for the cause of the CI failures, I don't think we can add strings and variables like this in jinja2 templates.

Maybe the extra href variable is not needed and everything can be put together in the line below?